packages/contracts/mud.config.ts

```diff
@@ -206,6 +206,8 @@ export default mudConfig({
       schema: {
         seatNum: "uint8",
         withPassword: "bool",
+        createdAtBlock: "uint64",
```

Comment:
I think `uint32` would be enough.

Reply:
With the growth of time and block generation rate, I think uint32 may not be sufficient at some point.